server/views/checkout.hbs

```diff
@@ -210,6 +210,21 @@
           Please select a valid country.
         </div>
       </div>
+      <div class="form-group form-check">
+        <input
+          type="checkbox"
+          name="bizumDemo"
+          class="form-check-input"
+          value="true"
+          id="bizumDemo"
+          {{#if details.bizumDemo}}checked{{/if}}
+          onchange="toggleBizumDemo({{ details.bizumDemo }})"
+        />
+        <label class="form-check-label" for="bizumDemo">
+          Generate a checkout less than 5€, this will allow Bizum to be used. Cofidis won't be available.
+          Notice that Bizum has to be enabled in your account.
```

Comment:
No need for this notice as this is a demo that uses our account. You should say something like: 

```
In test environment Bizum only supports transactions below 5 EUR. Generate a checkout less than 5€ to test Bizum.
```

Reply:
@jcuervas thats ok